retreats/2025.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: event
+title: Suspension Retreat 2025
+image: venues/studio_friction.jpg
+event_id: retreat2025
+---
+
+<br><br><br><br><br><br><br><br>
```

Comment:
Everything below here is "ToDo" to be included in the yaml + template.

If `_data/retreat2025/details.yml` starts to get too big, we can create a new file.

Maybe what's below here should go in `_data/retreat2025/important_info.yml`